samples/ClassroomLabs/Scripts/EthicalHacking/Create-EthicalHacking-LabsAccount.ps1

```diff
@@ -0,0 +1,111 @@
+[CmdletBinding()]
+param(
+    [parameter(Mandatory = $false)]
+    [string]$Email,
+
+    [parameter(Mandatory = $false, HelpMessage = "Default username for all VMs")]
+    [string]$Username = "AdminUser",
+
+    [parameter(Mandatory = $false, HelpMessage = "Default password for all VMs")]
+    [string]$Password = "P@ssword1!",
+
+    [parameter(Mandatory = $false, HelpMessage = "Default location for lab account")]
+    [string]$Location = "centralus",
+
+    [parameter(Mandatory = $false, HelpMessage = "Default location for lab account")]
+    [string]$ClassName = "EthicalHacking"
+)
+
+###################################################################################################
+#
+# Handle all errors in this script.
+#
+
+trap {
+    # NOTE: This trap will handle all errors. There should be no need to use a catch below in this
+    #       script, unless you want to ignore a specific error.
+    $message = $Error[0].Exception.Message
+    if ($message) {
+        Write-Host -Object "`nERROR: $message" -ForegroundColor Red
+    }
+
+    Write-Host "`nThe script failed to run.`n"
+
+    # IMPORTANT NOTE: Throwing a terminating error (using $ErrorActionPreference = "Stop") still
+    # returns exit code zero from the PowerShell script when using -File. The workaround is to
+    # NOT use -File when calling this script and leverage the try-catch-finally block and return
+    # a non-zero exit code from the catch block.
+    exit -1
+}
+
+###################################################################################################
+#
+# Main execution block.
+#
+
+# Install AzLab module
+Import-Module ..\..\Modules\Library\Az.LabServices.psm1 -Force
```

Comment:
I have mixed feelings about this.  I realize you did this because that is the path relative in this repo.  Maybe check if the module is already imported, then if not try to import from the location listed?

Reply:
One issue is the FORCE parameter. This forces the module to be updated to the latest version. Without that flag/parameter, there is a chance that you could be running an older version of this module.  I had this experience once Luca updated the Az.LabServices script and I didn't have this changes until I imported again with the force parameter.  This could be an issue for others if they have an older version imported.

Reply:
I've updated this to use the Invoke-WebRequest call in the script to download the Az.LabServices.psm1 script to install, and then the file after it's down installing. This will avoid the routing issue if you download the file manually instead of through git clone.